listenbrainz/webserver/static/js/src/FollowButton.tsx

```diff
@@ -187,5 +181,6 @@ class FollowButton extends React.Component<
     );
   }
 }
+FollowButton.contextType = APIContext;
```

Comment:
the react docs say that this can be added as [static field of the class](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties), which in my opinion is a bit easier to follow. Do we have this plugin?

Reply:
Not currently, I tried and got some errors. I can try adding the plugin though, I agree the static field syntax is better.